osquery/events/linux/http_event_publisher.cpp

```diff
@@ -0,0 +1,863 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include "http_event_publisher.h"
```

Comment:
The include order should be updated to match the following rules: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes